cli/tests/unit/http_test.ts

```diff
@@ -111,7 +111,7 @@ Deno.test(
 );
 
 Deno.test(
-  { permissions: { net: true } },
```

Comment:
@bartlomieju Will need help going through these ignored tests to make sure we are not missing a major issue.